Triangular_Number/Triangular_Number.c

```diff
@@ -0,0 +1,42 @@
+/*The nth triangular number is the number of dots in the triangular arrangement 
```

Comment:
Please mention the quadratic equation which provides the solution to this algorithm in the description section.